frontend/src/components/pages/RegistrantExperience/RegistrationSteps/Waiver/index.tsx

```diff
@@ -210,7 +210,7 @@ const WaiverPage = ({
                     });
                   }}
                   key={index}
-                  value={String(clause.optionSelected ? clause.agreed : null)}
+                  value={String(clause.agreed)}
```

Comment:
noting that any impossible state (ie. not 'true' or 'false' here) will highlight no buttons

```
Docs here - https://github.com/chakra-ui/chakra-ui/issues/2451
```